tests/test_tokenization_common.py

```diff
@@ -53,16 +53,28 @@ def merge_model_tokenizer_mappings(
     return model_tokenizer_mapping
 
 
-class TokenizerTesterMixin:
+DEBUG = False
```

Comment:
Why put it here? Looks a little weird to me. Maybe put it inside `TokenizerCommonTester`?

Reply:
I agree.

I was undecided where it'd be easier to see for someone looking to debug their code - close to where it's used, or on top of the file, where debug features should be normally turned on/off.